src/createMMKV.web.ts

```diff
@@ -21,6 +21,16 @@ export const createMMKV = (config: MMKVConfiguration): NativeMMKV => {
         'Tried to access storage on the server. Did you forget to call this in useEffect?'
       );
     }
+
+    try {
+      // throws ACCESS_DENIED error
+      window.localStorage;
+    } catch {
+      if (config.webFallbackStorage) {
+        return config.webFallbackStorage;
```

Comment:
re my prev comment; here would be the point where we construct the dummy `Map<K,V>` storage